### ISSUE TITLE
IMPORTANT: How to fix the input blocking bug

### ISSUE THREAD

**Issue by boriswinner** [xwpongithub/vue-range-slider#18]
Just comment these lines (1246, 1247) in the lib's js file:
`        addEvent(document, EVENT_KEY_DOWN, this.handleKeydown);`
`         addEvent(document, EVENT_KEY_UP, this.handleKeyup);`

If you have already written all the code and don't have the ability to switch to another library, this trick seems to do the job.

**Comment by chrismuiruriz** [comment]
Thank you for this tip

**Comment by chrismuiruriz** [comment]
Check out this https://stackoverflow.com/questions/60871872/vue-range-slider-making-page-not-scrollable-on-mobile

**Comment by AndriiBVW** [comment]
Remove installed package from npm:
`"vue-range-component": "^1.0.3"`

Add directly from Github
`"vue-range-component": "xwpongithub/vue-range-slider"`

**Comment by nielslynggaard** [comment]
Would be nice if the fix was just relased.

**Comment by asilcetin** [comment]
Yes, I have the same issue. It disables/blocks the keyboard input for the select input component (https://vue-select.org/) that I'm using. Please release the fix, thank you!

**Comment by mrfrkayvaz** [comment]
thanx 🙏

**Comment by nepster-web** [comment]
> Remove installed package from npm:
> `"vue-range-component": "^1.0.3"`
> 
> Add directly from Github
> `"vue-range-component": "xwpongithub/vue-range-slider"`

it seems this thread has been deleted?

**Comment by luisblazquezm** [comment]
> Remove installed package from npm: `"vue-range-component": "^1.0.3"`
> 
> Add directly from Github `"vue-range-component": "xwpongithub/vue-range-slider"`

Hands down to this 🙌 . Thanks!!
